resources/robots/magni/magni.json

```diff
@@ -0,0 +1,19 @@
+{
+    "magni":{
```

Comment:
Since we have multiple demo worlds in this repository with a different set of places, I'm wondering if we should add a `building_name` field that is the parent of "magni". So we will have two such fields, 1) `office` 2)airport_terminal

Reply:
You're right! I'll do that, Romi-dashboard will have a then a configuration file to know where to point at.

Reply:
Hi @Yadunund , I need to ask a question here, should an implementation need to support multiple worlds? For example, can the airport_terminal world contain an office world with different maps and different fleets and robots?

In case not:
I had been thinking and I realize that in the end game we will be referencing just one project (or world) in this case `office` or `airport_terminal`. I think these two worlds' resources should be on different folders and it should not have one file unifying its configurations. Each one should have its own main JSON configuration file and RomiDashboard should clone the world folder that we want to show.

Let me know what you think about it

Reply:
You're right, separating the main json files for different worlds is a better way to manage the files.